models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1567 @@
+| ``prior_inf_sd``               | Initial value of prior inflation standard deviation when      |
+|                                | ``write_prior_inf = .TRUE.``                                  |
++--------------------------------+---------------------------------------------------------------+               
+
+
+
+
+
+Step 11: Complete the Assimilation Setup
+---------------------------------------- 
+
+A few setup steps remain before the assimilation case can be executed.  First, the complete
+list of DART executables must be generated.  At this point you should have already customized
+your ``mkmf.template`` and tested your local build environment in Step 4.  In this step,
+you must compile the rest of the required DART scripts to perform the assimilation as follows:
+
+::
+
+ cd DART/models/clm/work/
+ ./quickbuild.csh -mpi
+
+After completion the following DART executables should be available within your ``work``
+folder.
+
+::
+
+  preprocess
+  advance_time
+  clm_to_dart
+  create_fixed_network_seq
+  create_obs_sequence
+  dart_to_clm
+  fill_inflation_restart
+  obs_diag
+  obs_seq_to_netcdf
+  obs_sequence_tool
+  filter
+  perfect_model_obs
+  model_mod_check
+
+Next modify the ``DART_params.csh`` settings such that the directories match
+your personal environment.  
+
+Modify the ``cesmtag`` and ``CASE`` variable:
+
+::
+ 
+ setenv cesmtag        <your cesm installation folder>
+ setenv resolution     f09_f09_mg17
+ setenv compset        2000_DATM%GSWP3v1_CLM50%BGC-CROP_SICE_SOCN_MOSART_SGLC_SWAV
+ setenv num_instances  5
+
+  ..
+  ..
+
+ if (${num_instances} == 1) then
+   setenv CASE clm5_f09_pmo_SIF
+ else
+   setenv CASE <your tutorial case name>
+ endif
+
+
+Modify the ``SourceModDir`` to match the directory that you set
+in Step 1:
+
+::
+
+ setenv use_SourceMods TRUE
+ setenv SourceModDir   <your SourceMods directory>
+
+
+Confirm the following variables are set to match your personal
+environment, especially ``cesmroot``, ``caseroot``, ``cime_output_root``,
+``dartroot`` and ``project``. 
+
+::
+
+ setenv cesmdata         /glade/p/cesmdata/cseg/inputdata
+ setenv cesmroot         /glade/work/${USER}/CESM/${cesmtag}
+ setenv caseroot         /glade/work/${USER}/cases/${cesmtag}/${CASE}
+ setenv cime_output_root /glade/scratch/${USER}/${cesmtag}/${CASE}
+ setenv rundir           ${cime_output_root}/run
+ setenv exeroot          ${cime_output_root}/bld
+ setenv archdir          ${cime_output_root}/archive
+ ..
+ ..
+ setenv dartroot         /glade/work/${USER}/git/DART_public
+ setenv baseobsdir       /glade/p/cisl/dares/Observations/land
+ ..
+ ..
+ setenv project      <insert project number>
+ setenv machine      cheyenne
+
+
+
+
+Step 12: Execute the Assimilation Run
+-------------------------------------
+
+Set up the assimilation case by executing ``CLM5_setup_assimilation``
+
+::
+
+ cd DART/models/clm/shell_scripts/cesm2_2/
+ ./CLM5_setup_assimilation
```

Comment:
- It would be good to mention how long the script is expected to take before completion (e.g., 5-10 mins). 
- Can we submit this script as a PBS job instead of just running it on the login node?  
- Sampling error correction is not used. Maybe we should not copy the SEC netcdf file to the run dir.